demo-terraform/aws/wordpress_ec2_alb_rds/alb.tf

```diff
@@ -1,53 +1,58 @@
 locals {
-    lb_name = "alb${random_string.lab_id.result}"
-    tg_name = "tg${random_string.lab_id.result}"
+  lb_name = "alb${random_string.lab_id.result}"
+  tg_name = "tg${random_string.lab_id.result}"
 }
 
 resource "aws_lb" "web" {
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure cross-zone load balancing is enabled for AWS LB (Load Balancer)**
 
**Vulnerability**
AWS Load Balancer has cross-zone load balancing disabled. This will prevent movement of traffic to registered targets outside the availability zone.
 
**Remediation**
In AWS Console - 
 1. Sign in to AWS Console and open EC2 console.
 2. On the navigation pane, under LOAD BALANCING, choose Load Balancers. 
 3. Select your load balancer. 
 4. On the Description tab, choose Change cross-zone load balancing setting. 
 5. On the Configure Cross-Zone Load Balancing page, select Enable. 
 6. Select Save.

source: accurics-